fish-select.js

```diff
@@ -24,6 +26,22 @@ const FishSelect = ({ visible, setVisible, selectedFish, onSelectFish }) => {
     { id: 'walleye', name: 'Walleye', image: require('./assets/fish/walleye.jpg') },
   ];
 
+  useEffect(()=>{
+    setLocationFish(undefined);
```

Comment:
why are we setting this as undefined like this, also why don't we use null? 
If they change location then change species, will it work